CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 This is the changelog of Grafeas server releases. For more information on
 versionining, see [versioning](docs/versioning.md) document.
 
+v0.1.5:
+  * Upgraded to golang 1.14.0
+  * Addeded last_scan_time to discovery occurrences
+  * Added support for Windows updates
+  * Changed resource type API Group to top level domain
+  * Added API config annotations
```

Comment:
What's this referring to?

Reply:
https://github.com/grafeas/grafeas/pull/404

Reply:
ditto here for dropping this from the changelog